Dockerfile

```
@@ -1,5 +1,5 @@
 # build stage: where we create binary
-FROM rust:1.66 AS builder
+FROM rust:1.77 AS builder
```

```suggestion
FROM rust:latest AS builder
```

어차피 밑에서 rustup 돌리니 그냥 latest로 가시지요